app/src/main/java/org/cru/godtools/viewmodel/GlobalDashboardDataModel.kt

```diff
@@ -0,0 +1,32 @@
+package org.cru.godtools.viewmodel
+
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.map
+import org.ccci.gto.android.common.db.Query
+import org.ccci.gto.android.common.db.getAsLiveData
+import org.cru.godtools.model.GlobalActivityAnalytics
+import org.keynote.godtools.android.db.GodToolsDao
+
+class GlobalDashboardDataModel(app: Application) : AndroidViewModel(app) {
+
+    private val globalAnalyticsQuery: LiveData<List<GlobalActivityAnalytics>>? by lazy {
+        Query.select(GlobalActivityAnalytics::class.java)
+            .getAsLiveData(GodToolsDao.getInstance(getApplication()))
+    }
```

Comment:
did you see my comment about using `dao.findLiveData()` instead of using a query?

Reply:
Sorry about that I will adjust that now.